packages/cli/src/models/files/ZosPackageFile.js

```diff
@@ -9,7 +9,24 @@ export default class ZosPackageFile {
 
   constructor(fileName = 'zos.json') {
```

Comment:
Note that this discovery should not apply to all commands. For instance, `zos init` should create a new project without doing any lookup. That said, this may not be the best place for this logic. Check if `ControllerFor` may work best, with a flag to prevent the lookup, used if called from zos init.

Reply:
It may also cause issues if we want to retrieve the list of contracts for a dependency. If the zos.json is missing, then this would climb up to the project root, and find the project's zos.json file.

Reply:
So I've added a commit that fixes your first point, if it is running an init command then it won't climb through directories looking for the zosfile. 
I'm not sure about the second point, though, how would a dependency in node_modules (which would have to have been installed by NPM) be broken enough to not have a zos.json file?